Source/themes.cpp

```diff
@@ -266,6 +266,8 @@ bool CheckThemeReqs(theme_id t)
 			rv = false;
 		}
 		break;
+	default:
+		break;

```

Comment:
```suggestion
		app_fatal("Invalid theme.");
```

Reply:
There are theme types that this does not apply